cpg_infra/driver.py

```diff
@@ -2522,6 +2523,18 @@ def setup_dataset_container_registry(self):
             member=self.analysis_group,
             membership=ContainerRegistryMembership.WRITER,
         )
+        self.infra.add_member_to_container_registry(
+            'test-full-reader-in-dev-container-registry',
```

Comment:
This should be 'test-full-reader-in-tmp-container-registry', otherwise pulumi won't create as new I think.